Classes/foxPlayerInput.uc

```diff
@@ -161,6 +162,8 @@ function ApplyWeaponFOV(Weapon Weap)
 {
 	local float ScaleFactor;
 
+	if (default.bCorrectWeaponFOV)
+	{

```

Comment:
I would change this to the following:
```suggestion
	//Abort if we don't want to correct visual weapon FOV
	if (!bCorrectWeaponFOV)
		return;

```
This simply avoids needing to wrap the entire function code in an "if" block, and helps keep the intended behavior obvious at a quick glance. 👍